src/Analysis/GWASWizard/wizard-endpoints/cohort-middleware-api.js

```diff
@@ -0,0 +1,102 @@
+import { useState, useEffect } from 'react';
```

Comment:
any particular reasons to keep the name of some of these files/directories not in camel cases?

Reply:
nope, camelCasedThem now